packages/name-resolution/src/sid/index.ts

```
@@ -1,72 +1,114 @@
+  ): Promise<string | null> {
+    if (isValidPaymentId(name)) {
+      console.log("PaymentId name", name, coint);
+      return this.handlePaymentIdGetAddress(name, paymentIdChain);
+    }
+
+    const tld = getTLD(name);
+    switch (tld) {
+      case "sol":
+        const solAddress = await this.solanaNameResolver.getAddress({
+          name,
+        });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)